SubmodelRepositoryServiceSpecification/V3.0_SSP-002.yaml

```diff
@@ -12,6 +12,7 @@ info:
     name: CC BY 4.0
     url: https://creativecommons.org/licenses/by/4.0/
   version: V3.0.1_SSP-002
+  x-profile-identifier: https://admin-shell.io/aas/API/3/0/SubmodelServiceSpecification/SSP-002
```

Comment:
Should be https://admin-shell.io/aas/API/3/0/SubmodelRepositoryServiceSpecification/SSP-002